docs/spring-boot-runtime/master.adoc

```diff
@@ -17,6 +17,10 @@ include::topics/sb-features-and-frameworks-summary.adoc[leveloffset=+2]
 
 include::topics/dev-guide-mission-intro.adoc[leveloffset=+1]
 
+[[spring-boot-tomcat-available-missions-and-boosters]]
+== {SpringBoot} - Available Missions and Boosters
```

Comment:
Replace _{SpringBoot} - Available_ with _{SpringBoot}--Available._ (You should use an em dash, which is represented by two hyphens in the source, instead of a hyphen surrounded by spaces.)